### PR TITLE
Made the api key private

### PR DESCRIPTION
**Pull request by HimanshuRanka**
It was only being used in one place so easy peasy lmnsqzy

### CODE REVIEW

File: components/ui/pathogen-map/pathogen-map.tsx [hunk 1]
```diff
@@ -142,7 +142,7 @@ export function PathogenMap<
       minZoom={2}
       maxZoom={14}
       interactiveLayerIds={[...layers.map((layer) => layer.id), 'country-highlight-layer']}
-      mapboxAccessToken={process.env.NEXT_PUBLIC_MAPBOX_API_KEY as string}
+      mapboxAccessToken={process.env.MAPBOX_API_KEY as string}
```
Comment (SeanKennyNF) [inline components/ui/pathogen-map/pathogen-map.tsx:145]: Vercel preview not working for me :(

image

Comment (SeanKennyNF) [reply components/ui/pathogen-map/pathogen-map.tsx:145]: We probably have to figure out something a bit more clever sadly :/

Comment (SeanKennyNF) [reply components/ui/pathogen-map/pathogen-map.tsx:145]: Oh also just checked vercel, I would def make sure this is set in production too.

Comment (HimanshuRanka) [reply components/ui/pathogen-map/pathogen-map.tsx:145]: yeah just saw that damn... ok will take a look

Comment (HimanshuRanka) [reply components/ui/pathogen-map/pathogen-map.tsx:145]: I guess it wasnt ezpz lmnsqzy

**Comment by HimanshuRanka** [comment]
This is not possible. Instead I had to just generate a public api key (this is safe to expose) and add url restrictions to it -- essentially from which urls it should accept requests etc. So just going to change the API key in vercel. The old api key still works and can be used on local anyway

**Pull request closed without merging**